packages/phone-number-privacy/common/src/utils/authentication.ts

```diff
@@ -42,25 +42,22 @@ export async function authenticateUser(
       // That is, it does not throw if the DEK is undefined or invalid
       logger.error({ err, shouldFailOpen }, ErrorMessage.FAILURE_TO_GET_DEK)
       if (shouldFailOpen) {
-        // TODO(2.0.0, release) add monitoring / alerting for these
-        logger.error(ErrorMessage.FAILING_OPEN)
-        return true
+        return { success: true, failedOpen: true }
```

Comment:
could you log the error directly here instead of passing `failedOpen` back to the `io` file? you could pass in the `warnings` and `config` variables as optional parameters to achieve the same effect